themes/dark/templates/config.mako

```diff
@@ -1,122 +1,5 @@
 <%inherit file="/layouts/main.mako"/>
```

Comment:
These files didn't get deleted on dark and light themes, need to manually remove